### PR TITLE
Update Gamepad status for Android Firefox and Chrome

### PR DESCRIPTION
**Pull request by luser**
Firefox for Android has had Gamepad API support since version 32: https://bugzilla.mozilla.org/show_bug.cgi?id=852935
Chrome for Android has had Gamepad API support since version 37: https://chromium.googlesource.com/chromium/src/+/41dada5c1a8036e5a335b209dfe9838503cf593f
```
$ git name-rev --tags 41dada5c1a8036e5a335b209dfe9838503cf593f
41dada5c1a8036e5a335b209dfe9838503cf593f tags/37.0.2027.0~116
```

### CODE REVIEW

File: features-json/gamepad.json [hunk 1]
```diff
@@ -219,10 +219,27 @@
...
     "and_chr":{
-      "44":"n"
+      "36":"n",
+      "37":"y",
+      "38":"y",
+      "39":"y",
+      "40":"y",
+      "41":"y",
+      "42":"y",
+      "43":"y",
```
Comment (cvrebert) [inline features-json/gamepad.json]: Data for older versions of Android Chrome & Android Firefox aren't currently accepted. See the Travis build failure.

Comment (luser) [inline features-json/gamepad.json]: Is there a note anywhere that describes this? Is it only valid to list the latest version there?

Comment (Fyrd) [inline features-json/gamepad.json]: @luser It's not really super-clear, but it is mentioned in https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md (you can't add new versions, can only modify values). So yes, only the latest version for mobile Chrome & Firefox right now. I'll update CONTRIBUTING.md to make that more clear.

**Comment by Fyrd** [comment]
Updated this manually.

**Comment by luser** [comment]
Thanks, I was going to take another look at this today. :+1:

**Pull request closed without merging**